Source/CarthageKit/Xcode.swift

```diff
@@ -199,21 +183,23 @@ public func locateProjectsInDirectory(directoryURL: NSURL) -> SignalProducer<Pro
 	let enumerationOptions: NSDirectoryEnumerationOptions = [ .SkipsHiddenFiles, .SkipsPackageDescendants ]
 
 	return NSFileManager.defaultManager().carthage_enumeratorAtURL(directoryURL.URLByResolvingSymlinksInPath!, includingPropertiesForKeys: [ NSURLTypeIdentifierKey ], options: enumerationOptions, catchErrors: true)
-		.reduce([]) { (var matches: [ProjectEnumerationMatch], tuple) -> [ProjectEnumerationMatch] in
-			let (enumerator, URL) = tuple
-			if let match = ProjectEnumerationMatch.matchURL(URL, fromEnumerator: enumerator).value {
-				matches.append(match)
+		.reduce([]) { (var matches: [ProjectLocator], tuple) -> [ProjectLocator] in
+			let (_, URL) = tuple
+			
+			if let UTI = URL.typeIdentifier.value {
```

Comment:
I can't say for sure. No tests are failing though. :trollface: 

But I'd rather simplify the code until and unless we find a failing case. Resolving symlinks is arguably a job for the consumer of `ProjectLocator`s anyway.